### PR TITLE
Initialize zustand store + jest

### PR DESCRIPTION
**Pull request by ErezBiren**
image

### CODE REVIEW

File: src/types.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export interface Question {
```
Comment (AmitGit217) [inline src/types.ts:1]: You can look at our back-end schemas and create interfaces from there

File: src/App.tsx [hunk 2]
```diff
@@ -1,34 +1,16 @@
...
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
+    <div>
+      <Navbar/>
+      <h1>{store.newQuestion}</h1>
```
Comment (AmitGit217) [inline src/App.tsx:11]: Cool so easy !

**Review by AmitGit217 — APPROVED** [review]
In general, I will approve all your PRs because you are the front-end leader but I will give my suggestions for you :)

File: src/types.ts (hunk 1 above)
Comment (ErezBiren) [reply src/types.ts:1]: Sure thing! please feel free to do that.

File: src/store.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+type Store = {
+  questions: Question[];
+  newQuestion: string;
+  addQuestion: () => void;
+};
+
+const useStore = create<Store>((set) => ({
+  questions: [],
+  newQuestion: "zustand is working",
+  addQuestion() {
```
[the remark this answers is not in the document]
Comment (ErezBiren) [reply src/store.ts:13]: Yes also thought about it. I'll explore that.

File: src/types.ts (hunk 1 above)
Comment (AmitGit217) [reply src/types.ts:1]: No problem I will create ts folder for all the base entities

Comment (ErezBiren) [reply src/types.ts:1]: I meant feel free to give your suggestions :)

**Pull request merged**